app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -193,6 +195,7 @@ private void setupRecorder() {
         }
 
         recorder.setAudioSource(MediaRecorder.AudioSource.MIC);
+        recorder.setAudioSamplingRate(RECORDING_SAMPLE_RATE);

```

Comment:
Is the audio sampling rate same for `HE_AAC` and `AMR_NB` ?

Reply:
Nop, [here](https://developer.android.com/guide/topics/media/platform/supported-formats#audio-formats) it says that `AMR_NB` should be sampled at 8kHz. In that scenario the audio will be downsampled to the appropriate rate before compression.

Reply:
Is it correct then to set the sampling rate to `44100` for both HE_AAC and AMR_NB ?

Reply:
@shubham1g5 I have seen in some places, but as suggested let's set the value according to the codec.